packages/strapi-parts/src/Dialog/Dialog.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Box } from '../Box';
+import { H2 } from '../Text';
+import { FocusTrap } from '../FocusTrap';
+import { Portal } from '../Portal';
+
+const DialogWrapper = styled.div`
+  position: absolute;
+  z-index: 3;
+  inset: 0;
+  // theme.colors.neutral800 with opacity
+  background: rgba(50, 50, 77, 0.2);
```

Comment:
We can have 8 bit hexa colors with opacity.

You can use a small helper to add opacity to the Hexadecimal code.
`const setOpacity = (hex, alpha) => ``${hex}${Math.floor(alpha * 255).toString(16).padStart(2, 0)}``;`